src/ChangesReporting/Output/ConsoleOutputFormatter.php

```diff
@@ -146,4 +178,23 @@ private function createSuccessMessage(ProcessResult $processResult, Configuratio
             $configuration->isDryRun() ? 'would have been changed (dry-run)' : ($changeCount === 1 ? 'has' : 'have') . ' been changed'
         );
     }
+
+    private function addEditorUrl(
+        string $filePath,
+        ?string $absoluteFilePath,
+        ?string $relativeFilePath,
+        ?string $lineNumber,
+    ): string {
+        $editorUrl = SimpleParameterProvider::provideStringParameter(Option::EDITOR_URL, '');
```

Comment:
This is the function that adds the link if the option is set, replacing the url template variables with their values